src/ray/gcs/gcs_server/test/gcs_server_rpc_test.cc

```diff
@@ -209,6 +257,31 @@ TEST_F(GcsServerTest, TestJobInfo) {
   TestMarkJobFinished(mark_job_finished_request);
 }
 
+TEST_F(GcsServerTest, TestNodeInfo) {
+  // Create gcs node info
+  ClientID node_id = ClientID::FromRandom();
+  rpc::GcsNodeInfo gcs_node_info = GenGcsNodeInfo(node_id.Binary());
+
+  // Register node info
+  rpc::RegisterNodeInfoRequest register_node_info_request;
+  register_node_info_request.mutable_node_info()->CopyFrom(gcs_node_info);
+  TestRegisterNodeInfo(register_node_info_request);
```

Comment:
`TestRegisterNodeInfo` is confusing. Just name it as `RegisterNodeInfo`.

Reply:
fixed